### PR TITLE
[Utopia-520] Updated Landing page to match design

### PR DESCRIPTION
**Pull request by antsand**
This commit includes couple of changes:
 - Converted header and footer styles to a class. Increase specificity
 - Added a wrapper class to make all elements align with header padding
 - Landing page has resposive elements and will somewhat adjust based on screen resolution

# Description

This PR includes the following proposed change(s):

- {List all the changes, if possible add the jira ticket #}

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Development Dependency Working Agreement
- [x] My code DOES NOT include the importing of new dependencies into the DPIA ecosystem
- [ ] My code DOES include the importing of new dependencies into the DPIA ecosystem
**If new dependencies are being introduced to the DPIA ecosystem:**
- [ ] The functionality of the dependency drastically reduces code complexity and makes my changes more easily maintainable and readible 
- [ ] The dependency being introduced does not contain multiple layers of nested dependencies introducing maintainability complexity to the DPIA ecosystem

## Frontend Development Changes
- [ ] N/A
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to project documentation or diagrams that reflect my changes
- [ ] New and existing unit tests pass locally with my changes
- [x] My code follows Airbnb React Style Guidelines

## API Development Changes
- [x] N/A
- [ ] I have performed a self-review of my own code
- [ ] My code follows standards and practices outlined in the BC Government API Development Guidelines
- [ ] New and existing unit tests pass locally with my changes
- [ ] My changes includes Swagger documentation updates that reflect the changes I am introducing

## Definition of Done

Definition of Done

### CODE REVIEW

File: src/frontend/src/pages/LandingPage/LandingPage.tsx [hunk 1]
```diff
@@ -17,8 +20,8 @@ function LandingPage() {
     win.location = `/${API_ROUTES.KEYCLOAK_LOGIN}`;
   };
   return (
-    <div>
-      <section className="hero-section">
+    <div class="">
```
Comment (binggaofw) [inline src/frontend/src/pages/LandingPage/LandingPage.tsx:23]: looks like you missing the class name here

**Comment by binggaofw** [comment]
if I check the landing page in a smaller screen, like my 16inch mbp, the text will mix with pic, i am not sure if this is the desire result
this is in 27inch monitor
Screen Shot 2022-11-24 at 2 26 43 PM
this is in small screen 
Screen Shot 2022-11-24 at 2 27 00 PM

**Comment by antsand** [comment]
> if I check the landing page in a smaller screen, like my 16inch mbp, the text will mix with pic, i am not sure if this is the desire result this is in 27inch monitor Screen Shot 2022-11-24 at 2 26 43 PM this is in small screen Screen Shot 2022-11-24 at 2 27 00 PM



> Good catch. We won't make changes to this till the designers tell us what it should look like for that resolution.

File: src/frontend/src/pages/LandingPage/LandingPage.scss [hunk 2]
```diff
@@ -46,10 +54,17 @@
...
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+  }
+    display: flex;
+    justify-content: space-between;
+    align-items: space-evenly;
+    padding-bottom: calc($padding-block + $footer-height);
+    //height: 710px;
```
Comment (kushal-arora-fw) [inline src/frontend/src/pages/LandingPage/LandingPage.scss:67]: commented code to be removed - or please add additional comments to justify the comment

File: src/frontend/src/sass/_variables.scss [hunk 3]
```diff
@@ -44,3 +44,23 @@ $header-height: 65px;
...
+  xs: 0,
+  sm: 576px,
+  md: 768px,
+  lg: 992px,
+  xl: 1200px,
+  xxl: 1400px,
+) !default;
+// scss-docs-end grid-breakpoints
+
+$wrapper-padding-inline:12em;
```
Comment (kushal-arora-fw) [inline src/frontend/src/sass/_variables.scss:66]: empty end of line missing

File: src/frontend/src/sass/_variables.scss [hunk 4]
```diff
@@ -44,3 +44,23 @@ $header-height: 65px;
 $navbar-height: 4rem;
 $padding-block: 7rem;
 $footer-height: 46px;
+
+
+/* Taken from bootstrap */
+// Grid breakpoints
+//
+// Define the minimum dimensions at which your layout will change,
+// adapting to different screen sizes, for use in media queries.
```
Comment (kushal-arora-fw) [inline src/frontend/src/sass/_variables.scss:53]: nit: can be changed to multi-line comment

File: src/frontend/src/sass/_variables.scss [hunk 5]
```diff
@@ -44,3 +44,23 @@ $header-height: 65px;
 $navbar-height: 4rem;
 $padding-block: 7rem;
 $footer-height: 46px;
+
+
```
Comment (kushal-arora-fw) [inline src/frontend/src/sass/_variables.scss:48]: extra line break

File: src/frontend/src/sass/_default.scss [hunk 6]
```diff
@@ -47,12 +47,14 @@ sub {
...
 
 .wrapper {
-  padding: 0 5px 0 5px;
   position: relative;
   max-width: 1286;
   margin-left: auto;
   margin-right: auto;
-  // height:100%;
+  padding-inline: $wrapper-padding-inline;
+  @include breakpoint(0 map-get($grid-breakpoints, "sm")) {
```
Comment (kushal-arora-fw) [inline src/frontend/src/sass/_default.scss:55]: consider adding a line break before the `@include`. I like it personally, see if it makes sense to you too.

File: src/frontend/src/pages/LandingPage/LandingPage.scss [hunk 7]
```diff
@@ -46,10 +54,17 @@
...
-  justify-content: space-evenly;
-  align-items: space-evenly;
-  height: 710px;
+  @include breakpoint(0 map-get($grid-breakpoints, 'xl')) {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+  }
+    display: flex;
```
Comment (kushal-arora-fw) [inline src/frontend/src/pages/LandingPage/LandingPage.scss:63]: indentation seems a bit off here

File: src/frontend/src/pages/LandingPage/LandingPage.scss [hunk 8]
```diff
@@ -1,14 +1,22 @@
 .hero-section {
   display: flex;
-  justify-content: space-around;
-  height: 952px;
-}
+  margin-top: calc($header-height);
+  min-height: calc(100vh - $header-height);
+  @include breakpoint(map-get($grid-breakpoints, 'xl')) {
```
Comment (kushal-arora-fw) [inline src/frontend/src/pages/LandingPage/LandingPage.scss:5]: consider adding line break before `@include`

File: src/frontend/src/components/common/Footer/_footer.scss [hunk 9]
```diff
@@ -1,28 +1,30 @@
...
   position: absolute;
   bottom: 0;
   left: 0;
   width: 100%;
-  height: $footer-height;
+  min-height: $footer-height;
+  display: flex;
+  align-items: center;
+  overflow:hidden;
   // top: calc(100vh - 72px);;
```
Comment (kushal-arora-fw) [inline src/frontend/src/components/common/Footer/_footer.scss:16]: remove comment if not needed - or can you please add comments justifying the presence.

**Review by kushal-arora-fw — APPROVED** [review]
Looks good now. Thank you!

**Review by binggaofw — APPROVED** [review]

**Pull request merged**